frontend_lib/src/component/Input/TextInput.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+require('./TextInput.styl')
+
+// A thin-layer around <input type='text'> that sets the Tracim style and behaviour
+// Also provide a onValidate handler which is called when Enter is pressed by the user.
+export class TextInput extends React.Component {
+  validateIfEnterKey(e) {
+    if (e.key !== 'Enter' || !props.onValidate) return
+    props.onValidate(e)
+  }
+
+  render() {
+    const { props } = this
+    return <div class='textinput__box'>
```

Comment:
to align vertically the opening and closing tags, we usually use parenthesis:
``` jsx
return (
  <div className=''>
    ...
  </div>
)

Reply:
also, `class` keyword is forbidden in jsx. you must use `className`.
The reason is that jsx is translated into js and the `class` keyword is a reserved js keyword